### DISCUSSION TITLE
How to enable previewjs extension with the preview links on vscode.

### DISCUSSION THREAD

**Discussion by MosheRivkin** [fwouts/previewjs#2320] [Q&A]
I've already installed the extension, yet, i don't see any link for the preview above the components. am i missing some steps?

![Screenshot 2023-12-19 142702](https://github.com/fwouts/previewjs/assets/88253247/e110fc09-2602-48cc-997d-659098f36345)
 
(i want it for svelte but i test it also on react)

**Comment by fwouts** [comment]
Hey @MosheRivkin, sorry for the late reply. If you go in the "Output" tab and select "Preview.js" in the dropdown on the right, what do you see?

**Reply by CastBart** [reply]
Hi, im experiencing the same problem. This is what i see in the output tab

![PreviewJS not working](https://github.com/user-attachments/assets/5ad19841-88b7-4395-a1eb-a3ccf21b9f3c)

**Reply by fwouts** [reply]
What package manager do you use? If you could provide an example repo that doesn't work, that'd be super helpful.

**Comment by icethorn** [comment]
I got similar problem and the output tab show
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/xxx/.vscode/extensions/zenclabs.previewjs-1.30.0/dist/node_modules/@previewjs/plugin-preact/dist/index.mjs' imported from /Users/xxx/.vscode/extensions/zenclabs.previewjs-1.30.0/dist/daemon.js

**Reply by LiamKarlMitchell** [reply]
Similar here.
```
 node --version
v23.11.0
✅ Detected compatible NodeJS version
🚀 Starting Preview.js daemon...
Streaming daemon logs to: c:\Users\Liam\.vscode-insiders\extensions\zenclabs.previewjs-1.30.1\dist\daemon-202504072106-55440.log
[21:06:48.708] FATAL (18064): Unable to load @previewjs/core from c:\Users\Liam\.vscode-insiders\extensions\zenclabs.previewjs-1.30.1\dist
Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'c:\Users\Liam\.vscode-insiders\extensions\zenclabs.previewjs-1.30.1\dist\node_modules\@previewjs\core\dist\index.mjs' imported from c:\Users\Liam\.vscode-insiders\extensions\zenclabs.previewjs-1.30.1\dist\daemon.js

```

I cd into the location and ran

1. yarn
2. Restarted vscode insiders

Now it seems to start the server.
```
$ node --version
v23.11.0
✅ Detected compatible NodeJS version
🚀 Starting Preview.js daemon...
Streaming daemon logs to: c:\Users\Liam\.vscode-insiders\extensions\zenclabs.previewjs-1.30.1\dist\daemon-202504110027-62497.log
[ready] Preview.js daemon server is now running at http://localhost:62497
```

However still no links to preview :(, I'm trying with solid js in a sub directory.
